lib/subspace/commands/ssh.rb

```diff
@@ -5,7 +5,8 @@ def initialize(args, options)
   end
 
   def run
-    ansible_command "ansible", @host, '-a', "/bin/bash -l"
+    exec "ssh ubuntu@#{@host}"
```

Comment:
The long term goal is to have it configured in ruby but this is a good idea for now